system-addon/content-src/components/TopSites/TopSite.jsx

```diff
@@ -153,7 +153,7 @@ class TopSite extends React.PureComponent {
               visible={isContextMenuOpen} />
           </div>
         }
-        {props.editMode &&
+        {props.onEdit &&
```

Comment:
Curious... does this work or break anything if we do
```js
{!props.onEdit ?
  <div>…
  :
  <div className="edit-menu">…
}
```

Eh.. I suppose the indenting might get wonky ;)

Reply:
hmmm. This passes linting:

```
        {!props.onEdit ?
          <div>
          ...
          </div> :
          <div className="edit-menu">
          ...
          </div>
        }
```

But it looks a little funny and the separation is less obvious. I'll leave it for now.

Reply:
Yeah, that's what I was afraid of. A small `:` at the end of the line.

Reply:
Although... oh well too late :p
```js
{!props.onEdit ? (
  <div>
    …
  </div>
) : (
  <div className="edit-menu"
   …
  </div>
)}
```

Eh.. still meh without changing a lot of indenting ;)